OsmAnd/src/net/osmand/plus/quickaction/actions/DayNightModeAction.java

```diff
@@ -25,14 +26,8 @@ public void execute(MapActivity activity) {
 				activity.getMyApplication().getSettings().DAYNIGHT_MODE.set(OsmandSettings.DayNightMode.NIGHT);
 				break;
 			}
-			case NIGHT: {
-				activity.getMyApplication().getSettings().DAYNIGHT_MODE.set(OsmandSettings.DayNightMode.DAY);
-				break;
-			}
-			case AUTO: {
-				activity.getMyApplication().getSettings().DAYNIGHT_MODE.set(OsmandSettings.DayNightMode.DAY);
-				break;
-			}
+			case NIGHT:
```

Comment:
Night, auto are switching to day which is strange. 
"DAY" (AUTO) -> "DAY" -> "NIGHT" [wrong]
"NIGHT" (AUTO) -> "DAY" -> "NIGHT" [correct]
"DAY" (AUTO) -> "NIGHT" -> "DAY" [correct]

Reply:
resolved